src/resources/postcss/components/your-tickets/_attendees.pcss

```diff
@@ -0,0 +1,34 @@
+.tec-tickets__attendees-list-order-attendees {
+	.event-tickets-wallet-plus__your_tickets_title_area {
```

Comment:
Anything that's ETWP related should live in ETWP. Is this part of the attendees list or is it relative to ETWP?

Also, we need to use hyphens and not underscores in the class names.